### PR TITLE
Translating Examples part 2 

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**
Simulate, interaction, objects, lights, motion, instance mode, dom examples translated.

An outcome of Google Summer of Code 2021
Author- Sanjay Singh Rajpoot @SanjaySinghRajpoot
Mentor - Aditya Rana @ranaaditya

### CODE REVIEW

**Review by SanjaySinghRajpoot — COMMENTED** [review]
typo checked

**Review by ranaaditya — APPROVED** [review]

File: src/data/examples/hi/10_Interaction/10_Tickle.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 /*
- * @name Tickle
- * @description The word "tickle" jitters when the cursor hovers over.
- * Sometimes, it can be tickled off the screen.
- */
+  * @name गुदगुदी
```
Comment (ranaaditya) [inline src/data/examples/hi/10_Interaction/10_Tickle.js]: Rename it to something more meaningful please !

**Review by ranaaditya — APPROVED** [review]

File: src/data/examples/hi/10_Interaction/10_Tickle.js (hunk 1 above)
Comment (SanjaySinghRajpoot) [reply src/data/examples/hi/10_Interaction/10_Tickle.js]: ok sir

Comment (SanjaySinghRajpoot) [reply src/data/examples/hi/10_Interaction/10_Tickle.js]: टिकल will be the right replacement all other keywords are similar to original word.

**Review by SanjaySinghRajpoot — COMMENTED** [review]
keyword replaced and checked

**Review by ranaaditya — APPROVED** [review]

**Pull request merged**